src/views/watershed/WatershedHanfordPlumes.vue

```diff
@@ -1,43 +1,87 @@
-      </p>
-      <p>
-      Proin mauris mauris, porttitor ac iaculis id, tristique vestibulum tortor. Sed placerat ac eros vestibulum pretium. Ut quis pharetra quam. Phasellus posuere, dolor ac convallis laoreet, mauris dui lobortis nisl, vel dictum est magna sit amet arcu. Ut ac egestas dui. Integer pulvinar nisi a nulla imperdiet rutrum id et est. Aliquam eget ultricies nisi. Nulla condimentum sollicitudin elit, a vehicula turpis consequat vel. Duis pharetra scelerisque dapibus. Maecenas bibendum volutpat arcu nec malesuada. Integer convallis auctor velit a facilisis. Cras sed molestie elit. Proin dui nibh, scelerisque at dui eget, feugiat consectetur risus. Etiam vel risus a elit tincidunt pulvinar. Maecenas et facilisis eros.
-      </p>
-      <p>
-      Donec varius elit lobortis ullamcorper tincidunt. Vestibulum lectus dolor, mattis vitae viverra at, auctor at lacus. Praesent maximus libero eget facilisis aliquam. Proin ac augue bibendum dolor ornare sagittis. Morbi ac lorem nunc. Donec malesuada id mi nec blandit. Aenean maximus molestie libero, non porta metus bibendum a. Aliquam rhoncus gravida aliquam. Mauris vitae dolor at nisi iaculis condimentum. Mauris non dolor nulla. Nunc euismod semper nisi. Maecenas vitae erat at lectus vehicula aliquet quis eget dolor. Nam molestie est in turpis efficitur tempus.
-      </p>
-      <p>
-      Morbi vitae ultricies risus. Morbi dapibus nibh eget ultricies venenatis. Nulla libero velit, porttitor id turpis non, luctus fringilla risus. Integer a nulla ut neque lacinia pellentesque. Aenean laoreet massa vel augue sollicitudin, non tristique sem porttitor. Vestibulum vitae nisi nunc. Sed in est luctus, posuere nisi aliquam, accumsan massa. Integer in ullamcorper felis. Nunc consequat diam eget sodales consequat. Nulla facilisi. Nunc aliquam quam ac nisl faucibus, in imperdiet magna volutpat. Vestibulum tristique enim magna. Aliquam erat volutpat. Maecenas consectetur augue a quam pharetra, in iaculis diam egestas. Proin non interdum massa.
-      </p>
+    <div class="title">
+      <h1>Body time</h1>
     </div>
-  </div>
-</template>
+    <div class="video">
+      <iframe width="350" height="197" src="https://www.youtube.com/embed/Sp_FVqKjZQQ?rel=0&amp;start=349;autoplay=0&mute=0&amp;showinfo=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+    </div>
+    <div class="caption">Testimony of Tom Bailie and Jay Mullen<br>recorded by <a href="https://www.youtube.com/watch?v=Sp_FVqKjZQQ" target="_blank">Paul DeRienzo</a></div>
```

Comment:
you should use the attribute/value `rel="nofollow"` whenever you use `target="_blank"`.